torch/_C/__init__.pyi.in

```diff
@@ -120,6 +120,7 @@ class _LegacyVariableBase(object):
 # Defined in torch/csrc/jit/python/init.cpp
 class Future(object):
   def __init__(self) -> None: ...
+  def done(self) -> _bool: ...
```

Comment:
@malfet is this the right way to annotate?

Reply:
Yes, thank you for adding the annotation.